row_run.py

```diff
@@ -3,20 +3,82 @@
 """
 the file to run to start the project
 """
+
+import logging
+from pathlib import Path
+from sys import stdout
+from os import environ
+from time import perf_counter
+import google.cloud.logging
+import google.cloud.storage
+import pandas as pd
+
 import row
 
-row.main()
-
-# images = row.download("bucket-name", 0)
-
-# for image in images:
-#     print(image)
-#     row.process(image)
-#     row.convert(image)
-#     row.rotate(image)
-#     row.prepare(image)
-#     row.detect(image)
-#     row.crop(image)
-#     row.prepare(image)
-#     row.detect(image)
-#     row.write(image)
+#: Set up logging
+logging.basicConfig(
+    stream=stdout,
+    format="%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s",
+    datefmt="%m-%d %H:%M:%S",
+    level=logging.DEBUG,
+)
+
+#: Set up variables
+URI_TEXTFILE = Path(__file__).parent / "bucket-info" / "udot_bucket_uris.txt"
+UDOT_BUCKET_NAME = "ut-udot-row-county-parcels"
+OUTPUT_BUCKET_NAME = "ut-dts-agrc-udot-parcel-recog"
+TASK_INDEX = environ["CLOUD_RUN_TASK_INDEX"]
+TASK_COUNT = environ["CLOUD_RUN_TASK_COUNT"]
+
+#: Set up main function
+def main():
```

Comment:
**quibble**: i think this could all be cleaned up with some row.py methods. This looks a bit too complicated for our main function.